### PR TITLE
Make the version of Python used by ARA configurable

### PR DESCRIPTION
**Pull request by Thulium-Drake**
Allow different distro's to use different version of the Python Interpreter for running venv commands

Fixes https://github.com/ansible-community/ara/issues/527

### CODE REVIEW

File: roles/ara_api/vars/CentOS_8.yaml [hunk 1]
```diff
@@ -41,3 +41,5 @@ ara_api_mysql_packages:
   - redhat-rpm-config
   - python36-devel
   - gcc
+
+ara_api_python_command: /usr/bin/python3.9
```
Comment (dmsimard) [inline roles/ara_api/vars/CentOS_8.yaml:45]: This should fix our issue with the jobs based on CentOS Stream 8 :+1: https://github.com/ansible-community/ara-collection/pull/71

File: roles/ara_api/tasks/database_engine/django.db.backends.mysql.yaml [hunk 2]
```diff
@@ -43,7 +43,7 @@
     name: mysqlclient
     state: present
     virtualenv: "{{ ara_api_venv | bool | ternary(ara_api_venv_path, omit) }}"
-    virtualenv_command: /usr/bin/python3 -m venv
+    virtualenv_command: "{{ ara_api_python_command }} -m venv
```
Comment (dmsimard) [inline roles/ara_api/tasks/database_engine/django.db.backends.mysql.yaml:46]:
```suggestion
    virtualenv_command: "{{ ara_api_python_command }} -m venv"
```

Comment (Thulium-Drake) [reply roles/ara_api/tasks/database_engine/django.db.backends.mysql.yaml]: Whoopsie :-) fixed!

**Comment by dmsimard** [comment]
Hi @Thulium-Drake, I've included your commit in https://github.com/ansible-community/ara-collection/pull/71 and merged it. You are the commit author: https://github.com/ansible-community/ara-collection/commit/ae5441b09ed61430bbe3f93e0d78b1fe4996780f

I will close this PR.

Thank you :)

**Pull request closed without merging**